src/aioauth/grant_type.py

```diff
@@ -79,8 +65,6 @@ def get_client_credentials(self, request: Request) -> Tuple[str, str]:
 
 
 class AuthorizationCodeGrantType(GrantTypeBase):
-    grant_type: GrantType = GrantType.TYPE_AUTHORIZATION_CODE
```

Comment:
I'm a little confused here. I might have missed this, but what is the reason for the removal of the `grant_type` class variable? Just out of curiosity.

Reply:
these checks were completely moved to the `AuthorizationServer` level:

https://github.com/aliev/aioauth/blob/e0b100fe687ef9940b769ff1da9ef3ad3eecb50b/src/aioauth/server.py#L129-L141